main.go

```diff
@@ -51,27 +51,30 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
+const componentName = "submariner-operator"
```

Comment:
Rather than redefining the component name, we can use the constant from https://github.com/submariner-io/submariner-operator/blob/ffeb0c063f9aafc0de6fe978d9b5564ed342fe89/pkg/names/names.go#L31